src/api/goods.ts

```diff
@@ -1,11 +1,13 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
+type Good = {
+  id: number
+  name: string
+  color: string
+};
+
 export function getAll(): Promise<Good[]> {
   return fetch(API_URL)
     .then(response => response.json());
 }
-
-export const get5First = () => {};
-
-export const getRedGoods = () => {};
```

Comment:
Don't delete this functions, you should use them in your code
On '5' button you should call get5First function, on 'red' button you should call getRedGoods function

Reply:
but these functions are the same